test/functional/editions_controller_test.rb

```diff
@@ -1433,6 +1433,57 @@ class EditionsControllerTest < ActionController::TestCase
     end
   end
 
+  context "#tagging_organisation_page" do
+    setup do
+      stub_linkables_with_data
+    end
+
+    context "user has govuk_editor permission" do
```

Comment:
We should rename this context block now that we have included a test for welsh editors (like is done for tagging mainstream browse page).
```suggestion
    context "user does not have editor permissions" do
```